src/main/resources/versions/1.7.2.json

```diff
@@ -0,0 +1,175 @@
+        },
+        {
+            "name": "org.lwjgl.lwjgl:lwjgl:2.9.0",
+            "rules": [
+                {
+                    "action": "allow"
+                },
+                {
+                    "action": "disallow",
+                    "os": {
+                        "name": "osx"
+                    }
+                }
+            ]
+        },
+        {
+            "name": "org.lwjgl.lwjgl:lwjgl_util:2.9.0",
+            "rules": [
+                {
+                    "action": "allow"
+                },
+                {
+                    "action": "disallow",
+                    "os": {
+                        "name": "osx"
+                    }
+                }
+            ]
+        },
+        {
+            "name": "org.lwjgl.lwjgl:lwjgl-platform:2.9.0",
+            "rules": [
+                {
+                    "action": "allow"
+                },
+                {
+                    "action": "disallow",
+                    "os": {
+                        "name": "osx"
+                    }
+                }
+            ],
+            "natives": {
+                "linux": "natives-linux",
+                "windows": "natives-windows"
+            },
+            "extract": {
+                "exclude": [
+                    "META-INF/"
+                ]
+            }
+        },
+        {
+            "name": "org.lwjgl.lwjgl:lwjgl:2.9.1-nightly-20131017",
+            "rules": [
+                {
+                    "action": "allow",
+                    "os": {
+                        "name": "osx"
+                    }
+                }
+            ]
+        },
+        {
+            "name": "org.lwjgl.lwjgl:lwjgl_util:2.9.1-nightly-20131017",
+            "rules": [
+                {
+                    "action": "allow",
+                    "os": {
+                        "name": "osx"
+                    }
+                }
+            ]
+        },
+        {
+            "name": "org.lwjgl.lwjgl:lwjgl-platform:2.9.1-nightly-20131017",
+            "rules": [
+                {
+                    "action": "allow",
+                    "os": {
+                        "name": "osx"
+                    }
+                }
+            ],
+            "natives": {
+                "osx": "natives-osx"
+            },
+            "extract": {
+                "exclude": [
+                    "META-INF/"
+                ]
+            }
+        },
+        {
+            "name": "net.java.jinput:jinput-platform:2.0.5",
+            "natives": {
+                "linux": "natives-linux",
+                "windows": "natives-windows",
+                "osx": "natives-osx"
+            },
+            "extract": {
+                "exclude": [
+                    "META-INF/"
+                ]
+            }
+        }
+    ],
+    "mainClass": "net.minecraft.launchwrapper.Launch",
+    "minimumLauncherVersion": 9ff
```

Comment:
Oops. I must have accidentally overwritten the last part. It should just be a `}`